### ISSUE TITLE
[DX] rework theme_pager()

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#797]
OMG [this](https://api.backdropcms.org/api/backdrop/core!includes!pager.inc/function/theme_pager/1) needs to be reworked.

**Comment by klonos** [comment]
Can you please elaborate @jenlampton? ...that is if you recall what annoyed you 😅

**Comment by BWPanda** [comment]
Probably the same thing that confused me: https://github.com/backdrop/backdrop-issues/issues/4697

**Comment by klonos** [comment]
🙄 😜 ...it does need simplification/rework.

**Comment by indigoxela** [comment]
As a general recommendation: a one-liner hardly ever provides enough information. We should (IMO) be a better role model regarding that. We expect others to provide more detailed issue descriptions, too. :smile:

**Comment by jenlampton** [comment]
Yes, sorry everyone about the useless issue text. Back in 2015 I was creating issues like crazy as 
"todo" items for myself more than anything else -- but apparently some are still lingering in the queue!

I've rewritten my original complaint to include an explanation and possible solution.
